CHANGELOG.md

```diff
@@ -2,14 +2,17 @@
 
 **This project is in the Beta stage. The API should be quite stable, but occasional breaking changes may be made.**
 
+### Breaking Changes
+* Fixed an issue that would cause passwords sent to the server (e.g. `Credentials.EmailPassword` or `EmailPasswordAuthProvider.registerUser`) to contain an extra empty byte at the end. (PR [#918](https://github.com/realm/realm-dart/pull/918)).
+  Notice: Any existing email users might need to be recreated because of this breaking change. 
```

Comment:
@nirinchev Added this notice here. Any good?

Reply:
Sure, sounds good